hikari/guilds.py

```diff
@@ -645,12 +645,12 @@ def icon_url(self) -> typing.Optional[files.URL]:
         """Icon for the guild, if set; otherwise `builtins.None`."""
         return self.format_icon()
 
-    def format_icon(self, *, format_: typing.Optional[str] = None, size: int = 4096) -> typing.Optional[files.URL]:
+    def format_icon(self, *, format: typing.Optional[str] = None, size: int = 4096) -> typing.Optional[files.URL]:
         """Generate the guild's icon, if set.
 
         Parameters
         ----------
-        format_ : typing.Optional[builtins.str]
+        format : typing.Optional[builtins.str]
```

Comment:
Should we be shadowing builtins?

Reply:
I changed this for consistency since some where using `format` and some where using `format_`. Renaming to `ext` would be the best, will do that :+1:

Reply:
if we're going to abbreviate it, is there any reason for preferring "ext" or "fmt"?

Reply:
Don't really care which one, up to you guys, I would say maybe `ext`